flow360/component/constants.py

```diff
@@ -3,7 +3,13 @@
 """
 
 
-class constants:
+class PhysicalConstants:
     """physical constants"""
 
     R = 287.0529
```

Comment:
It seems this `R` is not in use, we can remove
R is defined here: https://github.com/flexcompute/Flow360/blob/develop/flow360/component/flow360_params/physical_properties.py